experimental/origin-admin/src/pages/marketplace/mutations/CreateListing.js

```diff
@@ -214,7 +218,9 @@ class CreateListing extends Component {
                     />
                   </FormGroup>
                 </div>
-                <div style={{ flex: 1 }}>
+              </div>
+              <div style={{ display: 'flex' }}>
+                <div style={{ flex: 1, marginRight: 30 }}>
```

Comment:
I moved this to another row simply to make it and multi unit more readable, but I can put it back where it was.